app/models/patient.rb

```diff
@@ -871,6 +875,12 @@ def address_timezone_offset
     end
   end
 
+  # Check last_assessment_reminder_sent for eligibility. This is chiefly intended to help cover potential race condition of
+  # multiple reports being sent out for the same monitoree.
+  def last_assessment_reminder_sent_eligible?
+    last_assessment_reminder_sent <= 12.hours.ago || last_assessment_reminder_sent.nil?
```

Comment:
This will actually throw an error if `last_assessment_reminder_sent` is `nil`:
![Screen Shot 2021-01-07 at 4 58 31 PM](https://user-images.githubusercontent.com/11698457/103949644-93bb1900-5109-11eb-8ea3-eb826e1ec535.png)

Reply:
Good catch, swapped the checks so it will short circuit.